Svc/FpySequencer/FpySequencer.hpp

```diff
@@ -269,6 +375,29 @@ class FpySequencer : public FpySequencerComponentBase {
     //! Internal interface handler for directive_waitRel
     void directive_waitRel_internalInterfaceHandler(const FpySequencer_WaitRelDirective& directive) override;
 
+    //! Internal interface handler for directive_setLocalVar
+    void directive_setLocalVar_internalInterfaceHandler(const FpySequencer_SetLocalVarDirective& directive) override;
+
+    //! Internal interface handler for directive_goto
+    void directive_goto_internalInterfaceHandler(const Svc::FpySequencer_GotoDirective& directive) override;
+
+    //! Internal interface handler for directive_if
+    void directive_if_internalInterfaceHandler(const Svc::FpySequencer_IfDirective& directive) override;
+
+    //! Internal interface handler for directive_noOp
+    void directive_noOp_internalInterfaceHandler(const Svc::FpySequencer_NoOpDirective& directive) override;
+
+    //! Internal interface handler for directive_getTlmTime
+    void directive_getTlmTime_internalInterfaceHandler(const Svc::FpySequencer_GetTlmTimeDirective& directive) override;
```

Comment:
Recommendation: get full time + value and just value.  Time seems less useful on its own.  A separate call has a race condition.